modules/tinymce/src/plugins/table/main/ts/actions/InsertTable.ts

```diff
@@ -50,9 +51,15 @@ const insert = (editor: Editor, columns: number, rows: number, colHeaders: numbe
   editor.insertContent(html);
 
   return SelectorFind.descendant<HTMLTableElement>(Util.getBody(editor), 'table[data-mce-id="__mce"]').map((table) => {
+    const rawTable = table.dom();
     if (isPixelsForced(editor)) {
```

Comment:
While writing tests I found this had a bug where the enforcing only worked if we enforced `fixed` sizing. These changes should make sure all the modes are enforced.